java/src/Ice/src/main/java/com/zeroc/IceInternal/Instance.java

```diff
@@ -696,7 +698,7 @@ public void initialize(
 
     try {
       if (_initData.properties == null) {
-        _initData.properties = new com.zeroc.Ice.Properties();
+        _initData.properties = new Properties();
       }
 
       synchronized (Instance.class) {
```

Comment:
Like up here:
```suggestion
      synchronized (Instance.class) {
            Properties properties = _initData.properties;

```

Reply:
So that all the `_initData.properties` (which there are alot of) could just become `properties`.

Reply:
Yes, I've replaced a number of them.